### PR TITLE
fix spelling errors

### PR DESCRIPTION
**Pull request by whitewindmills**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix spelling errors

**Which issue(s) this PR fixes**:
none

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/lifted/selectors/bimultimap.go [hunk 1]
```diff
@@ -122,7 +122,7 @@ func (m *BiMultimap) Put(key Key, labels map[string]string) {
 			// No change to labels.
 			return
 		}
-		// Delete before readding.
+		// Delete before reading.
```
Comment (RainbowMango) [inline pkg/util/lifted/selectors/bimultimap.go:125]: This seems not read-ing, it's re-add-ing.

**Comment by Garrybest** [comment]
> Nice work! How do you find it? I mean do you find it by some tools?

+1, I wanna know as well. 😄

**Comment by whitewindmills** [comment]
@RainbowMango @Garrybest 
if possible, we can introduce `cspell` to Karmada pipelines.
https://github.com/streetsidesoftware/cspell

how to add it to Github Action?
https://www.youtube.com/watch?v=w8gGi3aeVpc

File: pkg/util/lifted/selectors/bimultimap.go (hunk 1 above)
Comment (whitewindmills) [reply pkg/util/lifted/selectors/bimultimap.go:125]: got it, misjudgment.

File: operator/pkg/tasks/init/cert.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func skipCerts(d workflow.RunData) (bool, error) {
 		return false, nil
 	}
 
-	if err := data.LoadCertFormSercret(secret); err != nil {
+	if err := data.LoadCertFormSecret(secret); err != nil {
```
[the remark this answers is not in the document]
Comment (whitewindmills) [reply operator/pkg/tasks/init/cert.go:49]: nice find

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Comment by RainbowMango** [comment]
> if possible, we can introduce cspell to Karmada pipelines.

That's interesting, I'm not sure it can handle/ignore `false positives`.

**Pull request merged**